csharp/test/Apache.Arrow.Adbc.Tests/DriverConnectionTests.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Apache.Arrow.Adbc.Tests
+{
+    /// <summary>
+    /// Abstract class for the ADBC connection tests.
+    /// </summary>
+    public abstract class DriverConnectionTests

```

Comment:
The Java/Go/C++ implementations have a generic test suite written in terms of the public API only along with hooks for individual drivers to inject connection options, etc. Without that I'm not sure there's value in purely a common interface for tests

Reply:
moved to a model where the base Adbc.Tests project does the validation, but the respective drivers are responsible for passing in the values to be validated